pkg/engine/k8s/api/v1alpha1/action_types.go

```diff
@@ -320,6 +320,18 @@ func (r *RenderingStatus) SetInputParameters(params []byte) {
 	r.Input.SetParameters(params)
 }
 
+// SetInputTypInstances sets the Action input TypeInstances.
+func (r *RenderingStatus) SetInputTypInstances(typeInstances []InputTypeInstance) {
```

Comment:
It's not directly connected with this, but please remove `CURRENTLY NOT IMPLEMENTED.` from https://github.com/capactio/capact/blob/de21b30a416a4d63152865d8bb713da516de6542/pkg/engine/k8s/api/v1alpha1/action_types.go#L278. Thanks!